tests/system.rs

```diff
@@ -297,29 +290,20 @@ fn handle_result(result: HashMap<String, String>,
 }
 
 // Map source file paths to their target paths.
-fn get_target(file_name: &str, target: Option<&str>, write_mode: Option<WriteMode>) -> String {
-    let file_path = Path::new(file_name);
-    let (source_path_prefix, target_path_prefix) = match write_mode {
-        Some(WriteMode::Coverage) => {
-            (Path::new("tests/coverage-source/"), "tests/coverage-target/")
+fn get_target(file_name: &str, target: Option<&str>) -> String {
+    if file_name.contains("source") {
+        let target_file_name = file_name.replace("source", "target");
+        if let Some(replace_name) = target {
+            Path::new(&target_file_name)
+                .with_file_name(replace_name)
+                .into_os_string()
+                .into_string()
+                .unwrap()
```

Comment:
This if branch does not work for all possible file names because it manipulates strings and not Paths, but I think this OK: we are not evil enough to create edge cases for ourselves =)